ansible_ai_connect/ai/api/views.py

```diff
@@ -770,10 +841,83 @@ def post(self, request) -> Response:
                 "format": "plaintext",
                 "generationId": generation_id,
             }
+
+        except ModelTimeoutError as e:
```

Comment:
We don't currently raise a ModelTimeoutError for playbook gen or explanation. Having a timeout on these new calls looks like a gap currently. Shall we remove this handling in both places for now for clarity?